### ISSUE TITLE
Document Kubernetes SD configuration.

### ISSUE THREAD

**Issue by fabxc** [prometheus/docs#163]
Once prometheus/prometheus#905 is merged.

**Comment by jimmidyson** [comment]
I'll get onto this in the next few days.

**Comment by juliusv** [comment]
@jimmidyson It'd be great to get this documented - IMO we should do a Prometheus release soon, and this will be blocking it :) Let me know if I can help!

**Comment by fabxc** [comment]
The authentication business is still bound to Kubernetes SD – not perfect but not blocking a release as long as the SD is flagged beta.

Some heavy testing wouldn't hurt before releasing.

**Comment by juliusv** [comment]
Yeah, it would be great to get that cleaned up before the next release, but not a total blocker if it would take too long.

**Comment by jimmidyson** [comment]
Ooo don't want to hold up next release :fearful: 

Aim to get it done tomorrow.

**Comment by juliusv** [comment]
@jimmidyson No worries, we probably won't do a (pre)release tomorrow, but perhaps sometime next week if the stars align properly.
